modules/insights-operator-what-information-is-collected.adoc

```diff
@@ -7,12 +7,15 @@
 
 The following information is collected by the Insights Operator:
 
-* General information about your cluster and its components to identify issues that are specific to your {product-title} version and environment
-* Configuration files, such as the image registry configuration, of your cluster to determine incorrect settings and issues that are specific to parameters you set
-* Errors that occur in the cluster components
-* Progress information of running updates, and the status of any component upgrades
+* General information about your cluster and its components to identify issues that are specific to your {product-title} version and environment.
+* Configuration files, such as the image registry configuration, of your cluster to determine incorrect settings and issues that are specific to parameters you set.
+* Errors that occur in the cluster components.
+* Progress information of running updates, and the status of any component upgrades.
 * Details of the platform that {product-title} is deployed on and the region that the cluster is located in
 ifndef::openshift-dedicated[]
-* Cluster workload information transformed into discreet Secure Hash Algorithm (SHA) values, which allows Red Hat to assess workloads for security and version vulnerabilities without disclosing sensitive details
+* Cluster workload information transformed into discreet Secure Hash Algorithm (SHA) values, which allows Red Hat to assess workloads for security and version vulnerabilities without disclosing sensitive details.
+* Workload information about the operating system and runtime environment, including runtime kinds, names, and versions.
+This gives Red Hat a better understanding of how you use {product-title} containers so that we can proactively help you make investment decisions to drive optimal utilization.
+For more information about the workload data that Red Hat collects by using this feature, see the documentation in the link:https://github.com/openshift/insights-operator/blob/fcb17ab1881135585e30d911ae5c44721a58e88a/docs/gathered-data.md#workloadinfo[Insights Operator GitHub repository].
```

Comment:
Do each of these sentences need to be on separate lines?

Reply:
Per the doc guidelines, we [cannot link to a repository that is hosted on www.github.com](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites).